cmd/query-frontend/main.go

```diff
@@ -23,8 +23,10 @@ func main() {
 			},
 		}
 		frontendConfig frontend.Config
+		maxMessageSize int
 	)
 	util.RegisterFlags(&serverConfig, &frontendConfig)
+	flag.IntVar(&maxMessageSize, "query-frontend.max-recv-message-size", 1024*1024*64, "Limit on the size of a grpc message this server can receive.")
```

Comment:
Would be good to mention the units, i.e, `size of a grpc message, in bytes, ...`